packages/spacecat-shared-data-access/test/unit/service/import-url/index.test.js

```diff
@@ -72,7 +72,7 @@ describe('Import Url Tests', () => {
           url: 'https://www.test.com',
         };
         await exportedFunctions.createNewImportUrl(mockImportUrl);
-        expect(mockDynamoClient.putItem.calledOnce).to.be.true;
+        expect(mockDynamoClient.putItem.calledOnce).to.equal(true);
```

Comment:
Same comment as above.

Reply:
The eslint check is failing - so, I had to modify the unit test.